components/translation.rst

```diff
@@ -29,9 +29,7 @@ catalogs*).
 Configuration
 ~~~~~~~~~~~~~
 
-The constructor of the ``Translator`` class needs one argument: The locale.
-
-.. code-block:: php
+The constructor of the ``Translator`` class needs one argument: The locale::
```

Comment:
@javiereguiluz @xabbuh that one will look a bit weird, won't it?

Reply:
I would stick with the explicit code block here

Reply:
> The problem with :: is that it looks strange and it's not easy to understand, whereas .. code-block:: php is beautiful and self-explanatory. Ping @xabbuh what do you think?

> I would stick with the explicit code block here

I'll extrapolate and take this as an agreement with @javiereguiluz 's feelings about `::` at the beginning of a line.

Reply:
But that probably means I should amend the standard.